### ISSUE TITLE
Thanks for you job.could you tell me the accuracy tested on OTB or vot datasets?

### ISSUE THREAD

**Issue by woailuoshenqi** [arbitularov/SiamRPN-PyTorch#5]

**Comment by arbitularov** [comment]
I have not finished the test section yet. When I finished, I will let you know (:

**Comment by willtwr** [comment]
For those who are curious.
I trained the model using this code and tested the performance using codes provided by foolwood.
The following are the AUC I got:
OTB2013: 0.5147
OTB2015: 0.4845

I don't think different training dataset will give this much difference.
So, I am suspecting either the codes are incompatible (between this repo and foolwood's) or hyper-parameter setting problem.
